engine/src/main/java/org/destinationsol/game/FarObject.java

```diff
@@ -16,19 +16,85 @@
 package org.destinationsol.game;
 
 import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.physics.box2d.Body;
 
+/**
+ * FarObject can be generally considered a {@link SolObject} simplified in terms of runtime code.
+ * <p>
+ * Generally speaking, every {@code SolObject} that gets too far away from player/camera, is transformed to a
+ * {@code FarObject} or deleted, with few specific exceptions. {@code FarObject}s then persist still in the game until
+ * the player/camera gets close to them again, at which point they are again transformed into their specific
+ * {@code SolObjects}. {@code FarObject}s generally have no body associated with them and do not inetract with the game much.
```

Comment:
Typo: "inetract"

On a side note I love how comparable this is to the Sector stuff in Terasology, if a fair bit simpler and easier to deal with :-)